investmentsFintech/investmentsFintech/AppDelegate.swift

```diff
@@ -17,16 +17,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         
         window = UIWindow(frame: UIScreen.main.bounds)
-        setupInitialModule()
+        checkInitialModule()
         
         return true
     }
     
     // MARK: - Private methods
     
-    private func setupInitialModule() {
-        let viewController = AuthModuleAssembly.assembleModule()
-        window?.rootViewController = viewController
+    private func checkInitialModule() {
+        
+        let storage = KeychainDataStorage()
+        let key = "MyAccountPassword"
```

Comment:
We use this key here and in `PINViewController.swift`. I think that this is not correct. Because we should remember all the places where this key is used and check that there is no typos in it. We should discuss this case